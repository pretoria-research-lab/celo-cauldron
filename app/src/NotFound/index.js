import React from 'react';

export default function NotFound() {

  return (<div className="container-fluid">
    <div className="row">
      <div className="col-lg-12 text-center">
        <h1 className="mt-5">Something went wrong :(</h1>
        <p className="lead">
          This resource does not exist
        </p>
      </div>
    </div>
  </div>)
}