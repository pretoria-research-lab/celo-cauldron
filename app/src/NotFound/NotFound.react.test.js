import React from "react";
import NotFound from "./index";
import renderer from "react-test-renderer";

test("NotFound component matches snapshot", () => {
	const component = renderer.create(
		<NotFound />,
	);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot(); 
});