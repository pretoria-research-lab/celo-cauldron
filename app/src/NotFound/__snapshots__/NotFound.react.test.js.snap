// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component matches snapshot 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-lg-12 text-center"
    >
      <h1
        className="mt-5"
      >
        Something went wrong :(
      </h1>
      <p
        className="lead"
      >
        This resource does not exist
      </p>
    </div>
  </div>
</div>
`;
